CHANGELOG

```diff
@@ -1,3 +1,7 @@
+==== 2.1.8 2019-02-14 ====
+
+* Fix a rare chance for a corrupt track file while writing metadate in out of disk situations. lp:1815305 
```

Comment:
metadat**a**

Reply:
Thank you. I have forced pushed this fix.